airflow/executors/base_executor.py

```diff
@@ -552,7 +554,12 @@ def terminate(self):
         """Get called when the daemon receives a SIGTERM."""
         raise NotImplementedError
 
-    def cleanup_stuck_queued_tasks(self, tis: list[TaskInstance]) -> list[str]:  # pragma: no cover
+    @deprecated(
```

Comment:
As this is a change of (public) API of BaseExecutor, are you back-porting this to `v2-10-test` branch (the Interface change) to know that other executor implementations need to adjust?

Otherwise, is this PR in general a bug-fix that would be valid to be ported to 2.10-line of Airflow or is this treated as new feature for Airflow 3? (I see this as 70% a bug fix for operational problem improvement and 30% a feature that myself I always found missing but never made it to the idea to contribute myself).

Reply:
i am not sure what this will be backported to.  I think 2.11 is the most "by the book" but others have suggested it could or should maybe go to 2.10 @potiuk @jedcunningham

Reply:
@dstandish Triued the new cherry-picker tool and made a back-port. feel free to debate/block/approve there.

I'd like to back-port, then the deprecation is made visible early. Additionally the missing feature (in my view: bug) is resolved in 2.10.4 as well.

Reply:
sounds good